tests/config/format-test.js

```diff
@@ -454,4 +451,13 @@ function format(originalText, originalOptions) {
   };
 }
 
+Reflect.defineProperty(runSpec, "prettier", {
+  get() {
+    return prettier;
+  },
+  set(prettierModule) {
+    prettier = prettierModule;
+  },
+  configurable: true,
+});
```

Comment:
Why use `Reflect.defineProperty`? I think `Object.defineProperty` is better for here because it throws `TypeError` error when it failed to define property.